firefly/README.md

```diff
@@ -0,0 +1,33 @@
+# Firefly
+
+## Overview
+Firefly is a Cloud Asset Management solution that helps Cloud teams to discover all their cloud footprint (incl. AWS, GCP, Kubernetes, and Datadog), automatically transform resources to Infrastructure-as-Code, detect drifts, and policy violations to make sure their cloud is aligned with its desired state.
+Firefly helps teams make their Datadog assets immutable, versioned, scalable and monitored by managing them as code in any Infrastructure-as-Code (IaC) tool of their choice.
+
+### Full Cloud Inventory
+Get a fully searchable inventory of all Datadog assets, alongside other cloud assets, such as AWS, K8s, GCP, Okta, and more.
+
+### Automatic Codification
+Automatically turn a single or multiple Datadog assets into code, including  Terraform, Pulumi, Cloudformation, and CDK.
```

Comment:
```suggestion
Automatically turn Datadog assets into code, including Terraform, Pulumi, Cloudformation, and CDK.
```

Reply:
Are Terraform, Pulumi, and Cloudformation examples of IaC tools that you can manage these assets in? I think some clarity on these examples would be helpful

Reply:
Firefly's ability is to describe DataDog assets as a code (Terraform / pulumi / cdk, etc) and manage them on Firefly.
The management itself of the resources is via Firefly and not solely through these tools as it applies in your phrasing

Reply:
No problem, we can keep this as-is.